docs/pages/access-controls/guides/per-session-mfa.mdx

```diff
@@ -218,37 +207,27 @@ spec:
   allow:
     node_labels:
       env: prod
-    kubernetes_labels:
-      env: prod
-    kubernetes_resources:
-      - kind: pod
-        namespace: "*"
-        name: "*"
-    db_labels:
-      'env': prod
-    db_users:
-    - '*'
-    db_names:
-    - '*'
+    logins:
+      - jerry
   deny: {}
 ```
 
 Olga then assigns both roles to all engineers, including Jerry.
 
-When Jerry logs into node `dev1.example.com` (with label `env: dev`), nothing
+When Jerry logs into node `jerry@dev1.example.com` (with label `env: dev`), nothing
```

Comment:
The node is still `dev1.example.com`, I'd revert that part of the change.

Reply:
Just trying to reinforce that `logins` is also relevant to the matching, how about this? 

a523bbac7577d749f1d6774c386ffb6885051e68